sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/ModuleIdentityLifecycleSamples.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System;
+using System.Threading.Tasks;
+using Azure.Iot.Hub.Service.Models;
+
+namespace Azure.Iot.Hub.Service.Samples
+{
+    /// <summary>
+    /// This sample goes through the lifecycle of a Module Identity for a device
+    /// </summary>
+    internal class ModuleIdentityLifecycleSamples
+    {
+        public readonly IoTHubServiceClient IoTHubServiceClient;
+        public const int MaxRandomValue = 200;
+        public readonly Random Random = new Random();
+
+        public ModuleIdentityLifecycleSamples(IoTHubServiceClient client)
+        {
+            IoTHubServiceClient = client;
+        }
+
+        public async Task RunSampleAsync()
+        {
+            string deviceId = $"device{Random.Next(MaxRandomValue)}";
+            string moduleId = $"module{Random.Next(MaxRandomValue)}";
+
+            // Create a DeviceIdentity.
+            DeviceIdentity deviceIdentity = await CreateDeviceIdentityAsync(deviceId);
```

Comment:
If we want to have good code snippets for the full crud experience, this is going to make it harder to do, much like with the code rewrite we're looking at having to do in ADT.

Reply:
We will be adding code snippets to the actual methods right? they should be self explanatory ... what do you recommend?

Reply:
In ADT we have lifecycle sample snippet, wouldn't it be same here?